main.go

```diff
@@ -81,15 +81,20 @@ func main() {
 
 	go func() {
 		Console := bufio.NewReader(os.Stdin)
+		service := support.Config.PossiblyService
 		for {
 			line, _, err := Console.ReadLine()
 			if err != nil {
+				if service {
+					return
+				}
 				support.ErrorLog(fmt.Errorf("%s: An error occurred when attempting to read the input to pass as input to the console\nDetails: %s", time.Now(), err))
```

Comment:
Maybe move that before service check to still have log entry?

Reply:
Notice that support.ErrorLog kills the whole application if called. So we need a new function that will only put the error into the file. 

But do we need that if the user specified that console will probably be unavailable?

Reply:
With some effort in service file (like I did) you can run service with tmux attached console so no point here. You can also make another flag for muting console output :shrug: